ts/features/newWallet/screens/WalletHomeScreen.tsx

```diff
@@ -81,6 +82,12 @@ const WalletScrollView = ({ children }: React.PropsWithChildren<any>) => {
     });
   };
 
+  const handleDeleteProfileButtonPress = () => {
+    navigation.navigate(ROUTES.NEW_PROFILE_STACK_NAVIGATOR, {
+      screen: ROUTES.NEW_PROFILE_DELETE_SCREEN
+    });
+  };
+
```

Comment:
What do you think to protect others flow with a local feature flag about the "new feature" newProfile? So if newFeatureProfile is enabled the app use the new feature.

Reply:
Ok sure! I will implement it now.

Reply:
A local feature flag (static) has been added to hide the new profile screen and the new profile delete flow.

Reply:
It's okay to insert the feature flag here. Note that this is about the remote feature flag. In some circumstances it is not necessary to use the remote one but instead use a local feature flag. Local FFs are env variables defined in config.ts file. https://github.com/pagopa/io-app/blob/15d61c58ef3001b516ddaea4391aa4b4c846c743/ts/config.ts#L98

Reply:
Thank you for pointing me to the correct file, I will move it there.